packages/wrangler/src/versions/deployments/index.ts

```diff
@@ -18,13 +18,13 @@ export default function registerVersionsDeploymentsSubcommands(
 	versionDeploymentsYargs
 		.command(
 			"list",
-			"Displays the 10 most recent deployments of your Worker [beta]",
```

Comment:
removing as the `beta` nature of these commands is signalled at command level, which should be enough?

Reply:
cc// @RamIdeas